examples/example_notebooks/getting_environment.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
+
+import gym_electric_motor as gem
+from gym_electric_motor.reference_generators import \
+    MultipleReferenceGenerator,\
+    WienerProcessReferenceGenerator
+from gym_electric_motor.visualization import MotorDashboard
+from gym.spaces import Discrete, Box
+from gym.wrappers import FlattenObservation, TimeLimit
+from gym import ObservationWrapper
+
+
```

Comment:
This is an auxillary file for the 'real' notebooks, right? If yes, we should give a small information hint at the beginning of the file to explain its overall purpose.

Reply:
this file is deleted, now